pending_code_changes.txt

```diff
@@ -1 +1,2 @@
-
+Code changes:
+ - Always use preferred intl prefix if present, not just for numbers with a non-unique IDD. This means we will output "8~10" as the prefix if calling formatOutOfCountryCallingNumber instead of "810" for some regions that have this tilde in their prefix [designates that the user should wait before continuing to dial].
```

Comment:
I would phrase this as:
"Changes formatOutOfCountryCallingNumber to always use ....." so people understand the affected method more.